projects/uci-console/src/lib/components/terms-conditions/terms-conditions.component.html

```diff
@@ -26,14 +26,14 @@ <h4><strong>Terms of Use</strong></h4>
                         "<strong>GoI</strong>").
                     </p>
                     <p>
-                        These terms of use, as amended, govern the usage of DIKSHA (Digital
+                        These terms of use, as amended, govern the usage of Nipun Lakshya App Chatbot (Digital
                         Infrastructure for Knowledge Sharing) by its Users (as defined below)
-                        ("<strong>Terms</strong>"). DIKSHA is an initiative of the GoI.
+                        ("<strong>Terms</strong>"). Nipun Lakshya App Chatbot is an initiative of the GoI.
```

Comment:
I see the name is changed from DIKSHA to Nipun ,the same can happen in future or we can re use the same code in another project.So for that we can create a function which takes the name as an argument and can return the final string like :
`const getText=(name:string):string=> 'some text {{name}} some other text' `